src/utils/charts/getChartData.js

```diff
@@ -36,32 +75,85 @@ getData.section8 = function (xRange, client, multiplier) {
   return data;
 };  // End getData.section8()
 
+/**
+ * @param {array} xRange All income values to be included.
+ */
+const insertBenefitData = function (xRange, clone, multiplier, benefitDatasets) {
+
+  // Don't loop if there's nothing to do
+  if (benefitDatasets.length === 0) {
+    return benefitDatasets;
+  }
+
+  // Otherwise, loop over incomes
+  for (var incomei = 0; incomei < xRange.length; incomei++) {
+    var income = xRange[ incomei ];
+    clone.future.earned = income / multiplier;
```

Comment:
We're talking it over in real-time since I think we're not on the same page. Regardless, I'm not sure I've expressed this well here, so it obviously needs to be clarified somehow.